buildenv/jenkins/variables/defaults.yml

```diff
@@ -84,8 +84,6 @@ build_discarder:
     OMR: 20
     OpenJDK: 20
     Personal: 50
-    # tmp PRs
-    pullRequest: 50
```

Comment:
Is it appropriate to remove this before converting PRs to pipeline builds?

Reply:
I _think_ this was never actually used. I believe I added it as part of #2138 in preparation for when this PR was delivered but it is now obsolete and replaced with #4738.

Reply:
#4738 must be a typo.

Reply:
#4783